LuluTest/element/alert_element.py

```diff
@@ -1,7 +1,24 @@
-from LuluTest.element import BaseElement
+from LuluTest.element import BaseElement, PageElement
 
 
 class AlertElement(BaseElement):
     def __init__(self, name=''):
         super().__init__(name)
         self.is_alert_element = True
+    @property
+    def is_page_element(self):
+      return True
+
+    @is_page_element.setter
+    def is_page_element(self, *args, **kwargs):
+      # This property cannot be changed
+      return False
+
+    @property
+    def is_alert_element(self):
```

Comment:
this class is the `AlertElement` class, so this method here needs to return `True`